ztp/ztp-policy-generator/Makefile

```diff
@@ -0,0 +1,33 @@
+PWD := $(shell pwd)
+BASE_DIR := $(shell basename $(PWD))
+
+# Keep an existing GOPATH, make a private one if it is undefined
+GOPATH_DEFAULT := $(PWD)/.go
+export GOPATH ?= $(GOPATH_DEFAULT)
+GOBIN_DEFAULT := $(GOPATH)/bin
+export GOBIN ?= $(GOBIN_DEFAULT)
+export PATH := $(PATH):$(GOBIN)
+XDG_CONFIG_HOME=./
+KUSTOMIZE_PLUGIN_HOME ?= $(XDG_CONFIG_HOME)/kustomize/plugin
+
+.PHONY: build test test-kustomize all clean
+build:
+	go build -mod=vendor -o policygenerator/PolicyGenerator ./policygenerator/
```

Comment:
Move the `-mod=vendor` out into the `$GOFLAGS` env so that other tools like `go test` also use it.

Example:
https://github.com/openshift-kni/cnf-features-deploy/blob/e7db36a82dc7e5194028b392a4cf00ecb53435cb/ztp/Makefile#L3